docs/_index.md

```diff
@@ -38,30 +38,28 @@ While `super` and its accompanying data formats are production quality, the proj
 a number of different elements of the system:
 * The [super data model](formats/zed.md) is the abstract definition of the data types and semantics
 that underlie the super-structured data formats.
-* The [super data formats](formats/_index.md) are a family of
+* The [super-structured data formats](formats/_index.md) are a family of
 [human-readable (Super JSON, JSUP)](formats/jsup.md),
 [sequential (Super Binary, BSUP)](formats/bsup.md), and
 [columnar (Super Columnar, CSUP)](formats/csup.md) formats that all adhere to the
 same abstract super data model.
-* The [SuperPipe language](language/_index.md) is the system's pipeline language for performing
+* [SuperSQL](language/_index.md) is the system's language for performing
 queries, searches, analytics, transformations, or any of the above combined together.
-* A  [SuperPipe query](language/overview.md) is a script that performs
-search and/or analytics.
-* A [SuperPipe shaper](language/shaping.md) is a script that performs
-data transformation to _shape_
-the input data into the desired set of organizing super-structured data types called "shapes",
-which are traditionally called _schemas_ in relational systems but are
-much more flexible in SuperDB.
+* A [SuperSQL pipe query (SPQ)](language/overview.md) is a query that
+employs SuperSQL's unique pipeline extensions and shortcuts to perform data
+operations that are difficult or impossible in standard SQL.
```

Comment:
I'd appreciate attention here that folks are on-board with this description. We want it to be clear to users that we'll be able to handle standard SQL queries, but we've got the pipeline extensions and shortcuts, so I've heard rough consensus around "SPQ" as the shorthand when talking about SuperSQL queries that include more than the standard stuff, and how it could also appear as a `.spq` extension for queries saved in files so a user knows they can't expect it contents to run seamlessly on some other SQL-based RDBMS without modification. When we get to filling out the intro language page more we'll surely go into detail, but I figure it's appropriate in the intro to plant the seed that we're offering more than standard SQL.